src/test/java/io/percy/selenium/SdkTest.java

```diff
@@ -108,4 +112,46 @@ public void snapshotWithOptions() {
     options.put("widths", Arrays.asList(768, 992, 1200));
     percy.snapshot("Site with options", options);
   }
+
+    @Test
+    public void takeScreenshotWhenNonRemoteWebDriver() {
```

Comment:
we have integration tests here ( which actually makes a build in percy) lets have that. 

for negative case, we can have unit tests.

Reply:
We have already discussed difficulty in writing integration test. and ninad suggested to mock remote web driver
https://browserstack.slack.com/archives/G01CRRW4YUT/p1683288765252299?thread_ts=1683277336.093029&cid=G01CRRW4YUT

Reply:
@itsjwala  the problem is we need a remote server and if we want to maintain that there are couple of issues. This was discussed in standup and hence we ended up with mocking this. Otherwise we will need to maintain a userName and accessKey which will establish a remote connection to run it. I guess either in one of the call/thread we discussed this and we decided to mock it as samar also did it at one particular place.

Reply:
Mocking should be fine